courses/machine_learning/text_classification/automl_for_text_classification.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>typo in SAMPLE_DATASET_NAME. Should be "titles_sample.csv"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done!</p>